src/applications/vaos/tests/services/location/index.unit.spec.js

```diff
@@ -0,0 +1,53 @@
+import { expect } from 'chai';
+import {
+  mockFetch,
+  setFetchJSONResponse,
+  setFetchJSONFailure,
+} from 'platform/testing/unit/helpers';
+
+import { getLocations } from '../../../services/location';
+import facilities983 from '../../../api/facilities_983.json';
+
+describe('VAOS Location service', () => {
+  describe('getLocations', () => {
+    let data;
+
+    it('should make successful request', async () => {
+      mockFetch();
+      setFetchJSONResponse(global.fetch, facilities983);
+      data = await getLocations({
+        systemId: 'var983',
+        parentId: 'var983A6',
+        typeOfCareId: '123',
+      });
+
+      expect(global.fetch.firstCall.args[0]).to.contain(
+        '/vaos/systems/983/direct_scheduling_facilities?type_of_care_id=123&parent_code=983A6',
+      );
+      expect(data[0].identifier[0].value).to.equal('urn:va:division:983:983');
+    });
+
+    describe('should return OperationOutcome error', async () => {
```

Comment:
Is this an `it`? 

Also not a blocker by any means, but do the front end specs ever use context? RSpec style it would still start with `describe('getLocations')` then context wraps the different cases like success vs error e.g. `context('with valid params')` or `context('with invalid params')` then the it `it('should make successful request')` or `it('should return OperationOutcome error')`.

It's purely semantics, the describe and context functions are identical, just helps how the specs read.

Reply:
Yeah, I think I copied this from another file before Narin fixed it. It should be describe.

We don't have a common pattern for grouping like that, though I agree that it can make tests easier to read.